src/workflow_components/composite_action.py

```diff
@@ -95,6 +99,7 @@ class CompositeAction(GraphObject):
     using = Property()
     image = Property()
     url = Property()
+    tag = Property()
```

Comment:
I think we talked about creating two fields? one for digest (to be unique), and one for list of tags?